content/minfraud/api-documentation/responses.md

```diff
@@ -1580,14 +1580,50 @@ Factors services, and a full example of the JSON body document for an error.
       "type": "military"
     },
     "risk_reasons": [
+  {
+    "multiplier": 45,
+    "reasons": [
       {
         "code": "ANONYMOUS_IP",
-        "reason": "The IP address belongs to an anonymous network. See /ip_address/traits for more details."
+        "reason": "The Anonymous IP address raised the overall risk score"
       },
       {
-        "code": "MINFRAUD_NETWORK_ACTIVITY",
-        "reason": "Suspicious activity has been seen on this IP address across minFraud customers."
+        "code": "IP_ISSUER_ID_NUMBER_VELOCITY",
+        "reason": "The number of distinct Issuer ID Numbers found in the 
+        velocity check on IP address raised the overall risk score"
```

Comment:
This is invalid JSON, that's why it is highlighted in red. There can't be a line break in the middle of a string. Is it possible to put it all on one line?

```suggestion
        "reason": "The number of distinct Issuer ID Numbers found in the velocity check on IP address raised the overall risk score"
```

Reply:
I think I understand that this is for formatting/display purposes. The problem is that if a developer copies this into a tool or something expecting valid JSON, it won't work.

If y'all want it displayed in a particular format, scotty can probably achieve that with CSS. I can create a linear issue for that purpose.